src/freenet/clients/fcp/ClientRequest.java

```diff
@@ -27,6 +27,9 @@
  */
 public abstract class ClientRequest implements Serializable {
 
+    /**
+     * ATTENTION: When incrementing this, please skip version 2. Version 2 had already temporarily
+     * been used by a development branch. */
```

Comment:
UID 2 had been in use by the initial version of this pull request, the commit to remove it was only added after some time.
I had announced the pull request on IRC, and thus other people might have been using it already.
So I'd rather be save than sorry. There are more than enough version UIDs, its a long :)